src/language-graphql/printer-graphql.js

```diff
@@ -33,43 +33,17 @@ function genericPrint(path, options, print) {
         hasOperation && !hasName && isNonEmptyArray(node.variableDefinitions)
           ? " "
           : "",
-        isNonEmptyArray(node.variableDefinitions)
-          ? group([
-              "(",
-              indent([
-                softline,
-                join(
-                  [ifBreak("", ", "), softline],
-                  path.map(print, "variableDefinitions")
-                ),
-              ]),
-              softline,
-              ")",
-            ])
-          : "",
+        printVariableDefinitions(path, print),
         printDirectives(path, print, node),
-        node.selectionSet ? (!hasOperation && !hasName ? "" : " ") : "",
+        !hasOperation && !hasName ? "" : " ",
```

Comment:
We have not tests for operation definitions that don't have `selectionSet`. Is it possible?

Reply:
https://github.com/prettier/prettier/issues/13401

Reply:
It seems `selectionSet` is required https://github.com/graphql/graphql-js/blob/8be83d8d528991aed04ca17434b7e26e56f649cf/src/language/ast.ts#L343